packages/marshal/src/encodeToCapData.js

```diff
@@ -117,7 +115,7 @@ export const makeEncodeToCapData = (encodeOptions = {}) => {
    * Readers must not care about this order anyway. We impose this requirement
    * mainly to reduce non-determinism exposed outside a vat.
    *
-   * @param {Passable} passable
+   * @param {any} passable
```

Comment:
it was `any` before because `Passable` was `any`. Making it `Passable` now causes an error at just about every access in this function. The job of this function is to inspect what the object actually is, narrow the type, and delegate. So the type hint isn't hindering more than helping.

Reply:
Thx. Resolving.

Reply:
Would `unknown` work better than `any` here, or is it better to blanket-ignore the type-unsafety internally?